app/i18n/de.json

```diff
@@ -92,28 +92,28 @@
         "TYPE": "Typ"
     },
     "NODE": {
-        "NOT_REACHABLE_MSG": "Cluster ist nicht erreichbar",
+        "NOT_REACHABLE_MSG": "Kein Cluster erreichbar",
```

Comment:
I question this translation:
Kein Cluster erreichbar = No cluster available
Cluster nicht erreichbar = Cluster not available